doc/manual/src/language/syntax.md

```diff
@@ -183,6 +183,9 @@ This section covers syntax and semantics of the Nix language.
   Numbers, which can be *integers* (like `123`) or *floating point*
   (like `123.43` or `.27e13`).
 
+  Integers in the Nix language are 64-bit signed integers.
+  Integer overflow is defined to throw an error.
```

Comment:
```suggestion
```

This page is about the syntax, so I don't think we need to duplicate this here.
We have a link on the next line for the semantics.